index.js

```diff
@@ -139,6 +139,7 @@ class K8sVMExecutor extends Executor {
             container: config.container,
             api_uri: this.ecosystem.api,
             store_uri: this.ecosystem.store,
+            metric_host: hoek.reach(this.ecosystem, 'metric_host', { default: '' }),
```

Comment:
Could you add jsdoc for `this.ecosystem.metric_host`? https://github.com/screwdriver-cd/executor-k8s-vm/blob/769f8d639df540c680960daa37456056735feb07/index.js#L64